<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GRVisualization</class>
 <widget class="QMainWindow" name="GRVisualization">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GR SDK</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="GRGLWidget" name="GLwidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>431</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2">
         <widget class="QTreeWidget" name="devicesTree">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>500</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(88, 88, 88);</string>
          </property>
          <column>
           <property name="text">
            <string>Connected GR Devices</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QWidget" name="settings" native="true">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>200</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>130</width>
            <height>250</height>
           </size>
          </property>
          <widget class="QCheckBox" name="trajectoryCheckBox">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>10</y>
             <width>89</width>
             <height>22</height>
            </rect>
           </property>
           <property name="text">
            <string>Trajectory</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QPushButton" name="randomData">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>150</y>
             <width>109</width>
             <height>30</height>
            </rect>
           </property>
           <property name="maximumSize">
            <size>
             <width>240</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>RandomData</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="rightHandCheckBox">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>50</y>
             <width>97</width>
             <height>22</height>
            </rect>
           </property>
           <property name="palette">
            <palette>
             <active>
              <colorrole role="Button">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>59</red>
                 <green>2</green>
                 <blue>51</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="Button">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>59</red>
                 <green>2</green>
                 <blue>51</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="Button">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>59</red>
                 <green>2</green>
                 <blue>51</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Right Hand</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QCheckBox" name="loggingCheckBox">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>130</y>
             <width>78</width>
             <height>22</height>
            </rect>
           </property>
           <property name="text">
            <string>More</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QCheckBox" name="hackerModeCheckBox">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>110</y>
             <width>109</width>
             <height>22</height>
            </rect>
           </property>
           <property name="text">
            <string>Hacker Mode</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="leftHandCheckBox">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>70</y>
             <width>88</width>
             <height>22</height>
            </rect>
           </property>
           <property name="text">
            <string>Left Hand</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QCheckBox" name="pausePlotCheckBox">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>90</y>
             <width>89</width>
             <height>22</height>
            </rect>
           </property>
           <property name="text">
            <string>Pause Plot</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="rotationsCheckBox">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>30</y>
             <width>89</width>
             <height>22</height>
            </rect>
           </property>
           <property name="text">
            <string>Rotations</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTabWidget" name="Plots">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_all_data">
        <attribute name="title">
         <string>All Sensors</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QCustomPlot" name="accelerometer" native="true">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCustomPlot" name="gyroscope" native="true"/>
         </item>
         <item>
          <widget class="QCustomPlot" name="magnetometer" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_accelerometer">
        <attribute name="title">
         <string>Accelerometer</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QCustomPlot" name="only_accelerometer" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>300</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_gyroscope">
        <attribute name="title">
         <string>Gyroscope</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QCustomPlot" name="only_gyroscope" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>300</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_magnetometer">
        <attribute name="title">
         <string>Magnetometer</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCustomPlot" name="only_magnetometer" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>300</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionWhat">
   <property name="text">
    <string>&amp;what?</string>
   </property>
  </action>
  <action name="actionIs">
   <property name="text">
    <string>&amp;is</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GRGLWidget</class>
   <extends>QWidget</extends>
   <header>GRGLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
